utils/dependencies.rb

```diff
@@ -1,17 +1,23 @@
 require 'rubygems'
+require 'io/console'
+
+def bundle_install_prompt
+  bundle_check = `bundle check`.chomp
+  return unless bundle_check != "The Gemfile's dependencies are satisfied"
 
-bundle_check = `bundle check`.chomp
-if bundle_check != "The Gemfile's dependencies are satisfied"
   puts bundle_check
-  puts 'Install? [Yn]'
-  if gets.chomp == 'n'
+  print 'Install? [Yn] '
+  if $stdin.getch == 'n'
     puts 'Exiting...'
     exit
   end
+  puts 'Installing...'
 
   puts `bundle install`
 end
 
+bundle_install_prompt
```

Comment:
It looks okay, but do we actually need to extract these lines to a method?

Reply:
I replaced the method wrapping for just a comment explaining what the part of the code does. Thanks for the review!